vizro-core/src/vizro/models/_components/button.py

```diff
@@ -32,8 +31,4 @@ class Button(VizroBaseModel):
 
     @_log_call
     def build(self):
-        return html.Div(
-            dbc.Button(id=self.id, children=self.text, className="button_primary"),
-            className="button_container",
-            id=f"{self.id}_outer",
-        )
+        return dbc.Button(id=self.id, children=self.text)
```

Comment:
This sort of tidying is great and I think a sign we're doing the right thing 👍 The more we can do this, the better. When I compare the DOM to what it used to look like on VizX or even when we first released Vizro, it's so much cleaner now and that is great. We should definitely continue pushing in this direction for nice simple, semantic HTML without unnecessary nesting. 💯